src/main/java/com/vinylteam/vinyl/dao/VinylDao.java

```diff
@@ -0,0 +1,19 @@
+package com.vinylteam.vinyl.dao;
+
+import com.vinylteam.vinyl.entity.Vinyl;
+
+import java.util.List;
+
+public interface VinylDao {
+    void saveUniqueVinyls(List<Vinyl> uniqueVinyls);
+
+    void saveVinyls(List<Vinyl> vinyls);
+
+    List<Vinyl> getUniqueVinyls();
+
+    List<Vinyl> getVinyls();
+
+    Vinyl getUniqueVinylById(long id);
+
+    Vinyl getVinylById(long id);
+}
```

Comment:
Maybe we don't need SaveVinyls if we have VinylDao - saveMany, getAll, getAllUnique, getUniqueById, getById..?
@AlexKievDeveloper

Reply:
Do I understand correctly what is meant to rename methods?